gneiss/stats/composition.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from skbio import DistanceMatrix
+
+
+def variation_matrix(X):
+    """ Calculate Aitchison variation matrix.
+
+    Parameters
+    ----------
+    X : np.array
```

Comment:
I get a 'Data must be symmetric and cannot contain NaNs.' error if providing the following example data: 
```
X = pd.DataFrame(np.array([[ 1,8,7,2,2],
                           [10,5,3,2,0],
                           [ 4,4,4,4,4],
                           [ 6,4,0,4,6]]).T)
```
Do you expect that #samples == #features ? It's not clear from the docstring